ext/kv/remote.rs

```diff
@@ -272,6 +272,7 @@ impl<P: RemoteDbHandlerPermissions> Database for RemoteDb<P> {
     &self,
     _state: Rc<RefCell<OpState>>,
   ) -> Result<Self::QMH, AnyError> {
+    eprintln!("Deno.Kv.listenQueue is not supported for remote KV databases");
```

Comment:
We print colored warnings in the CLI (like [this one](https://github.com/denoland/deno/blob/cd0fcc2257198e33c319555091b7081dc8387c8d/cli/args/mod.rs#L602)) - maybe we should do the same thing here?

Reply:
`colors` module is in `deno_runtime` crate. @bartlomieju - would it be ok to bring in `deno_runtime` dependency here?

Reply:
That might be a bit problematic as it would cause a circular dependency between the two. Maybe I suggest directly embedding an ansi escape code here, alternatively copy-pasting a bit of code from `runtime/colors.rs`?